README.md

```diff
@@ -19,6 +19,47 @@ Or direct from the repository:
 
 ```pip install https://github.com/easyScience/EasyScience```
 
+### Development with Pixi
```

Comment:
Consider keeping the README concise and instead linking to CONTRIBUTING.md, where this information is already documented.